packages/hint-manifest-scoped-urls/tests/tests.ts

```diff
@@ -0,0 +1,148 @@
+import { test } from '@hint/utils';
+import { HintTest, testHint } from '@hint/utils-tests-helpers';
+
+const { generateHTMLPage, getHintPath } = test;
+const hintPath = getHintPath(__filename);
+
+const htmlWithManifestSpecified = generateHTMLPage('<link rel="manifest" href="site.webmanifest">');
+
+const tests: HintTest[] = [
+    {
+        name: 'No start_url property specified in Manifest file',
+        reports: [{ message: `Property 'start_url' not found in manifest file` }],
+        serverConfig: {
+            '/': htmlWithManifestSpecified,
+            '/site.webmanifest': { content: `{}` }
+        }
+    },
+    {
+        name: 'Manifest property start_url not scoped.',
+        reports: [
+            {
+                message: `'start_url' is not in scope of the app.`,
+                position: { match: 'start_url": "/",' }
+            }
+        ],
+        serverConfig: {
+            '/': htmlWithManifestSpecified,
+            '/site.webmanifest': {
+                content: `{
+                    "short_name": "Test webhint App",
+                    "start_url": "/",
+                    "scope": "/test"
+                }`
+            }
+        }
+    },
+    {
+        name: 'Manifest property start_url is relative and inaccessible.',
+        reports: [
+            {
+                message: `Specified 'start_url' is not accessible. (status code: 404).`,
+                position: { match: 'start_url": "randomcontent",' }
+            }
+        ],
+        serverConfig: {
+            '/': htmlWithManifestSpecified,
+            '/site.webmanifest': {
+                content: `{
+                    "short_name": "Test webhint App",
+                    "start_url": "randomcontent",
+                    "scope": "/"
+                }`
+            }
+        }
+    },
+    {
+        name: 'Manifest property start_url is not same origin',
+        reports: [
+            {
+                message: `'start_url' must have same origin as the manifest file.`,
+                position: { match: 'start_url": "https://example.com",' }
+            }
+        ],
+        serverConfig: {
+            '/': htmlWithManifestSpecified,
+            '/site.webmanifest': {
+                content: `{
+                    "short_name": "Test webhint App",
+                    "start_url": "https://example.com",
+                    "scope": "/"
+                }`
+            }
+        }
+    },
+    {
+        name: 'Manifest property start_url is absolute and scoped but inaccessible',
+        reports: [
+            {
+                message: `Specified 'start_url' is not accessible. (status code: 404).`,
+                position: { match: 'start_url": "/test",' }
```

Comment:
Technically this is a "root-relative" URL, not an absolute URL. An absolute URL starts with a protocol (e.g. `https://`). 

As a side note, there's even a fun intermediate called a "protocol-relative" URL (e.g. `//mysite.com`), but I wouldn't worry about that one. 😉

Reply:
Nice to know 😊 

Changed it to `start_url` with preceding `/`